tomls/conflux.toml

```diff
@@ -0,0 +1,21 @@
+[contract]
+# The contracts the poller needs to interact with
+wrb = "0x8BE1a41D04b0298AcE4F10263605154e7A980980"
+# Addresses of BtcUsdPriceFeed, EthUsdPriceFeed and GoldEurPriceFeed contracts
+pricefeeds = ["0x8B637aBd43B9d7d6714DC03eef0bA410a77eAE98",
+              "0x89Fca84D6570b88f478d2c72677CAC6e32B01e26"]
```

Comment:
There are only 2 price feeds but in the comment you mention 3 :smile:

Reply:
Corrected.